CONTRIBUTING.md

```diff
@@ -27,4 +27,29 @@ Custom Information - if you're copying this template for the first time you can
 
 ## Contact Information
 
-- [Slack channel](https://kubernetes.slack.com/messages/k8s-code-organization) 
\ No newline at end of file
+- [Slack channel](https://kubernetes.slack.com/messages/k8s-code-organization) 
+
+## PR Workflow
```

Comment:
optional - you can also add a link to https://www.kubernetes.dev/docs/guide/github-workflow/ for more info on the GitHub workflow